### ISSUE TITLE
Besteht die Möglichkeit den Landing Screen "Conneting to Server / Loading values" farblich den Systemeinstellungen anzupassen?

### ISSUE THREAD

**Issue by MichaelSchaaf999** [ioBroker/ioBroker.vis-2#41]
**Is your feature request related to a problem? Please describe.**
Wie @JB_Sullivan schon im Forum kommentierte, Zitat:
[@foxriver76](https://forum.iobroker.net/uid/5950) Besteht die Möglichkeit den Landing Screen "Conneting to Server / Loading values" farblich den Systemeinstellungen des Smartphone anzupassen?

Derzeit ist er ja immer grell weiß, was gerade bei schwachem Umgebungslicht, dem Blick in eine Taschenlampe gleich kommt - das ist leider nicht sehr Augenfreundlich.

Wäre schön, wenn er auch die (dunklen) Systemeinstellungen des Smartphone annehmen könnte.

Ergänzt um:
[@jb_sullivan](https://forum.iobroker.net/uid/8907) [@foxriver76](https://forum.iobroker.net/uid/5950) : Das ist auch bei uns seeeehr störend, auf allen Geräten, auch auf den sechst Kiosk-iPads an der Wand - diese "grelle Taschenlampe" ist unschön. Am liebsten wäre es, wenn man einen Farbparameter irgendwo in den Projekteinstellungen mitgeben könnte.

**Describe the solution you'd like**
Farbauswahl des zur Zeit nur weißen Hintergrundes.

**Describe alternatives you've considered**
-

**Additional context**
-

**Comment by Feuer-sturm** [comment]
@MichaelSchaaf999 mit vis-2 version 2.2.2 is zumindest schon einmal der Ladescreen auch dunkel, wenn ich im Editor den Dark mode einstelle.

**Comment by MichaelSchaaf999** [comment]
@Feuer-sturm: Welche Einstellung ist gemeint: Diese hier?
![image](https://github.com/ioBroker/ioBroker.vis-2/assets/53481090/bab83494-dcf3-4fc9-91a3-663a0e6e485b)

**Comment by Feuer-sturm** [comment]
@MichaelSchaaf999 Die von dir gezeigte Einstellung hat bei mir keine Auswirkung, so dass hier immer der helle Ladescreen erscheint. Es wäre aber auch meine Erwartung, dass es über die Checkbox funktioniert.

Wenn ich hier den Darkmode vom Editor einstelle, dann habe ich auch eine dunkle Ladescreen Seite am Rechner
![grafik](https://github.com/ioBroker/ioBroker.vis-2/assets/25422320/f2f06506-4804-41c8-b5d5-6033674b3f3a)

**Comment by MichaelSchaaf999** [comment]
Ja, bestätigt: Checkbox funktioniert (noch) nicht, Darkmode im Editor funktioniert.

**Comment by Jacksson5** [comment]
Das ist ja Interessant... es funktioniert mit diesem Trick aber leider nur im Browser auf dem PC, im FullyKiosk auf dem Tablet lädt es weiterhin mit dem weißen Bildschirm 😕.
Gibt es da eine Möglichkeit das dort auch dunkel zu bekommen?
